### DISCUSSION TITLE
Vercel IP range for serverless functions

### DISCUSSION THREAD

**Discussion by nathanchu** [vercel/vercel#5504] [Help] [answered]
What IP or IP Ranges will serverless functions use? I'm wondering so I can allow them on mongodb atlas. Is it just 76.76.21.21? Or is that just a reverse proxy?

**Comment by styfle** [comment, marked as answer]
There is no defined range at this time.

Take a look at this article for more info: https://vercel.com/knowledge/how-to-allowlist-deployment-ip-address
